tests/ImageSharp.Benchmarks/Codecs/DecodeTiff.cs

```diff
@@ -1,38 +1,94 @@
 // Copyright (c) Six Labors.
 // Licensed under the Apache License, Version 2.0.
 
+//// #define BIG_TESTS
+
 using System.IO;
+
 using BenchmarkDotNet.Attributes;
+
+using SixLabors.ImageSharp.Formats.Experimental.Tiff;
 using SixLabors.ImageSharp.PixelFormats;
 using SixLabors.ImageSharp.Tests;
+
 using SDImage = System.Drawing.Image;
 using SDSize = System.Drawing.Size;
 
 namespace SixLabors.ImageSharp.Benchmarks.Codecs
 {
-    [Config(typeof(Config.ShortClr))]
+    [Config(typeof(Clr))]
     public class DecodeTiff : BenchmarkBase
     {
-        private byte[] tiffBytes;
+        public class Clr : Config.ShortClr
+        {
+            public Clr() => this.SummaryStyle = BenchmarkDotNet.Reports.SummaryStyle.Default.WithMaxParameterColumnWidth(60);
+        }
+
+        private string prevImage = null;
+
+        private byte[] data;
+
+        private Configuration configuration;
+
+#if BIG_TESTS
```

Comment:
I would rather not use the `#if BIG_TESTS` here and only use the bigger images or why do you think there is a need for both? Does it take too long?

Reply:
ok i, think i can answer my one question here: I understand now why those images are called big 😄 
I was not aware that those images are generated. The uncompressed images are more than 700MB, thats way to much.

I would suggest to use the `Calliphora_x` images in the `Tiff` folder for the benchmarks. With 804x1198 pixels they are reasonable big and we do not need the script to generate the images then.

Reply:
Yes, the files is very big and takes too long. For the repository they are large, but big files are actually for tiff. It's such a compromise.  I'll look `Calliphora_x`  files

Reply:
What's the resolution here? I can still see the conditional in place (now unused)

Reply:
ah yes, this case used for local stress testing, what way can you suggest to correctly implement this?

Reply:
Lets leave it as is then for now